gef.py

```diff
@@ -3257,6 +3304,213 @@ def del_setting(self, name):
 #     def do_invoke(self, argv):
 #         return
 
+@register_command
+class PieCommand(GenericCommand):
+    """PIE breakpoint support"""
+
+    _cmdline_ = "pie"
+    _syntax_  = "{:s} (breakpoint|info|delete|run|attach|remote)".format(_cmdline_)
+
+    def __init__(self):
+        super(PieCommand, self).__init__(prefix=True)
+        return
+
+    def do_invoke(self, argv):
+        if len(argv)==0:
+            self.usage()
+            return
+
+
+@register_command
+class PieBreakpointCommand(GenericCommand):
+    """Set a PIE breakpoint"""
+
+    _cmdline_ = "pie breakpoint"
+    _syntax_  = "{:s} BREAKPOINT".format(_cmdline_)
+
+    def do_invoke(self, argv):
+        global __pie_counter__, __pie_breakpoints__
+        if (len(argv) < 1):
+            self.usage()
+            return
+        bp_expr = ' '.join(argv)
```

Comment:
Check for single quotes, you added a lot, but also use double quotes. Let's be consistent.